src/api/track.ts

```diff
@@ -160,15 +160,41 @@ class TrackAPI {
           sclVal,
           cslVal,
         };
+
+        let { user, events, token, expiresAt } = response;
+        const location = { latitude, longitude, accuracy };
+        let passed;
+        let expiresIn;
+        if (expiresAt) {
+          expiresAt = new Date(expiresAt);
+          passed = user?.fraud?.passed && user?.country?.passed && user?.state?.passed;
+          expiresIn = expiresAt ? (expiresAt.getTime() - new Date().getTime()) / 1000 : null;
```

Comment:
nit: It looks like `expiresAt` will also be present here.

nit: You can also do `Date.now()` instead of `new Date().getTime()`

Reply:
Updated. @tderouin if we can expose `expiresIn` (in seconds, `expiresAt - now`) in the API response, I can also just remove this, along with `passed`

Reply:
https://github.com/radarlabs/server/pull/5823/commits/a8fd054e1f02f5e2b9a1c44f9cde6f44927db03c